cmake/YarpInstallationHelpers.cmake

```diff
@@ -89,8 +89,14 @@ type \"@_type@\"
 
   set(_path "${CMAKE_BINARY_DIR}/${${YCPI_INSTALL_VARS_PREFIX}_DYNAMIC_PLUGINS_INSTALL_DIR}") # (build tree)
   configure_file("${_in_file}" "${_build_file}" @ONLY)
+  if(WIN32)
+    string(REPLACE "/" "\\" _path ${_path})
+  endif()
```

Comment:
@claudiofantacci do you remember why you added this command? As `_path`  is overwritten in the following line, it is not actually doing anything.